Assets/Scripts/ChargingPig.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ChargingPig : EnemyController
+{
+    private bool animationPlayed;
+    private float animationDelay = 1.5f;
```

Comment:
Assuming these belong to the starting / notice animation, they should be named as such (startingAnimationPlayed) to clear ambiguity